mytools.py

```diff
@@ -6,7 +6,7 @@
 #
 
 
-import sys, os, time, string, cv2, PIL
+import sys, os, time, string, cv2, PIL.Image
```

Comment:
In line 504(old)/516(new) PIL.Image is used. You change PIL import to PIL.Image import. Will it work?

Reply:
the plain PIL import didn't work for me, but PIL.Image does